torch_xla/csrc/device.cpp

```diff
@@ -89,4 +89,16 @@ bool UseVirtualDevice() {
 
 bool GetLockSpmdConfig() { return spmd_config_is_locked; }
 
+bool CheckTpuDevice(XlaDeviceType hw_type) {
```

Comment:
Should we merge these two checks?

Reply:
Yes, given the purpose of original check, `hw_type == XlaDeviceType::TPU` I think this makes sense.